.github/workflows/webui-periodic.yml

```diff
@@ -0,0 +1,111 @@
+# Run webui e2e tests periodically in Permian
+name: webui-periodic
+on:
+  #schedule:
+  #  - cron: 0 23 * * *
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  scenario:
+    name: Webui periodic
+    runs-on: [self-hosted, kstest]
+
+    timeout-minutes: 65
+    env:
+      # ocp-master-xxl: 32GB RAM / 4GB RAM per VM
+      TEST_JOBS: 8
+      # The timeout should be a few minutes less then the job's timeout-minutes
+      # so that we get partial results and logs in case of the timeout.
+      LAUNCHER_TIMEOUT_MINUTES: 60
+      # URL to the unpacked installation image
+      # TODO: use preview image
+      STATIC_REPO_URL: "http://10.43.136.2/users/rv/s2/webui-zveleba-f39"
+
+    steps:
+      # self-hosted runners don't do this automatically; also useful to keep stuff around for debugging
+      # need to run sudo as the launch script and the container create root/other user owned files
+      - name: Clean up previous run
+        run: |
+          sudo podman ps -q --all --filter='ancestor=kstest-runner' | xargs -tr sudo podman rm -f
+          sudo podman volume rm --all || true
+          sudo rm -rf *
+
+      # The test library is in anaconda repository
+      - name: Clone anaconda repository
+        uses: actions/checkout@v3
+        with:
+          ref: master
+          fetch-depth: 0
+          path: anaconda
+
+      # TODO: use main branch when the webui workflow is merged there
+      - name: Clone Permian repository
+        uses: actions/checkout@v3.3.0
+        with:
+          repository: rhinstaller/permian
+          path: permian
+          ref: devel
+
+      - name: Clone tplib repository
+        uses: actions/checkout@v3.3.0
+        with:
+          repository: rhinstaller/tplib
+          path: tplib
+
+      # Use static URL to unpacked image
+      # TODO: unpack the image as the part of the workflow
+      - name: Set the image to be tested
+        id: prepare_repo
+        run:
+          echo "repo_url=$STATIC_REPO_URL" >> $GITHUB_OUTPUT
+
+      - name: Create Permian settings file
+        working-directory: ./permian
+        run: |
+          cat <<EOF > settings.ini
+          [AnacondaWebUI]
+          anaconda_repo=https://github.com/rhinstaller/anaconda.git
+          # TODO resolve using csbuild and pin to newer version
+          cockpit_branch=283
```

Comment:
This is now also default, and when newer version can be used it should be changed in the default settings

Reply:
So, where would I go to find the default?

Reply:
Defaults for this plugin can be found here [permian/libpermian/plugins/anaconda_webui/settings.ini](https://github.com/rhinstaller/permian/blob/devel/libpermian/plugins/anaconda_webui/settings.ini). We have documentation WIP https://github.com/rhinstaller/permian/pull/64 that talks about it and I will link it somewhere when it is done.